src/Controllers/ResultController.php

```diff
@@ -37,24 +37,36 @@ public function getAllUserTest(RequestInterface $request)
 
```

Comment:
This method renders two views. Both prefixed with admin-.
Then you check if the role is admin. 
Well if you render two views both prefixed with admin, I would expect the role to always be Admin. 
There is to much logic here and the method is confusing. 
Please refactor.